### DISCUSSION TITLE
How to free RAM memory after clear lazy object is loaded into RAM

### DISCUSSION THREAD

**Discussion by nhthong2001** [microstream-one/microstream#526] [Q&A] [answered]
Hello MicroStream team,

After I load a lazy object onto Ram, then I use Lazy.clear() to clear this object. How can I release the memory occupied by this lazy object immediately or as soon as possible to avoid OOM problem for loading other lazy objects onto Ram.

**Comment by hg-ms** [comment, marked as answer]
It’s important that there are no more references to lazy loaded objects in your code so that they can deleted by the java GC from memory. You may trigger the Java-GC trigger manually to force memory cleanup but that’s usually not necessary.
